tests/test_level/test_level_04.py

```diff
@@ -572,6 +572,19 @@ def test_print_list_access_index_var(self):
             extra_check_function=check_in_list
         )
 
+    def test_list_access_misspelled_at_gives_error(self):
+        code = textwrap.dedent("""\
+        dieren is Hond, Kat, Kangoeroe
+        print dieren at random""")
```

Comment:
This should be "ad" I think? Not sure why this test was not failing, it should (since the error is not raised). I do get an error locally so maybe something is up with the tests?